reana/reana_dev/release.py

```diff
@@ -101,4 +108,73 @@ def release_pypi(ctx, component, timeout):  # noqa: D301
         click.secho(f"{component} successfully released on PyPI", fg="green")
 
 
+@click.option(
+    "--version", "-v", help="REANA version to release, e.g. 0.7.0a1",
+)
+@click.option("--user", "-u", default="reanahub", help="DockerHub user name [reanahub]")
+@release_commands.command(name="release-helm")
+@click.pass_context
+def release_helm(ctx, version, user):  # noqa: D30
+    """Release REANA as a Helm chart.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+    """
+
+    def _update_values_yaml(new_docker_images):
+        """Update all images in ``values.yaml``, skipping the ones up to date."""
+        update_images_cmds = []  # FIXME
+        for docker_image in new_docker_images:
+            for cmd in update_images_cmds:
+                run_command(cmd, "reana")
+
+    remaining_docker_releases = []
+    new_docker_images = []
+    for component in REPO_LIST_CLUSTER:
+        if not is_component_dockerised(component):
+            continue
+
+        if not is_last_commit_tagged(component):
+            remaining_docker_releases.append(component)
+        else:
+            new_docker_images.append("")  # FIXME build the full docker+tag string
+
+    if remaining_docker_releases:
+        line_by_line_missing_releases = "\n".join(remaining_docker_releases)
+        click.secho(
+            "The following components are missing to be released:\n"
+            f"{line_by_line_missing_releases}",
+            fg="red",
+        )
+
+    if not is_last_commit_tagged("reana"):
+        click.echo("Nothing to be released")
+        sys.exit(0)
+
+    _update_values_yaml(new_docker_images)
+    version = bump_component_version(
+        "reana",
+        get_current_component_version_from_source_files("reana"),
+        next_version=version,
+    )
+
+    ctx.invoke(git_create_release_commit_command, component=["reana"], version=version)
```

Comment:
I wonder whether these two should be part of the `release-helm` command.  E.g. making PRs is not part of `release-pypi` and `release-docker` commands, so perhaps let's not use them here as well? That said, the first command to create release commit is perhaps good to keep, but creating the PR is perhaps too much; need local testing first?